src/apps/sources/__test__/redux/actions.test.ts

```diff
@@ -78,7 +78,7 @@ describe("action", () => {
     expect(indexedAction).toHaveBeenCalledWith("sources/retrieveSources", 1);
   });
 
-  it("for public sources should use action index as 2 along with the corresponding actionType and api", () => {
+  it("for All Public Concepts should use action index as 2 along with the corresponding actionType and api", () => {
```

Comment:
Don't replace this text... We only need things to read "All Public Concepts" in the UI or where we're testing something about the UI.